### PR TITLE
Draft of a server-side prep script for publish via SSH

### PR DESCRIPTION
**Pull request by hanke**

### CODE REVIEW

**Comment by hanke** [comment]
@Alexqw Can you please have a look at this script? It should become as portable as possible. Thanks!

File: tools/sshserver_prepare_for_publish.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+#
```
Comment (yarikoptic) [inline tools/sshserver_prepare_for_publish.sh]: copyright/license statement?

File: tools/sshserver_prepare_for_publish.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# a number of useful status messages on the target via stdout.
+#
+# Usage: ./sshserver_prepare_for_publish /tmp/example democol ds1 ds2
+#
+
+set -e
+set -u
+
+# TODO:think about different modes to tune accessibility of the published repos
+ROOT_PATH=$1
```
Comment (yarikoptic) [inline tools/sshserver_prepare_for_publish.sh]: "$1" here and below

**Comment by yarikoptic** [comment]
why bother with a script (take care about versioning, making sure that cmdline args aren't too many to fit in 1 invocation, etc) and not just drive necessary commands through ssh while maintaining persistent connection?

**Comment by hanke** [comment]
Persistent connection it will be. Whether we run it as this very script, or the same commands otherwise, I don't care. It still had to be explored.

File: tools/sshserver_prepare_for_publish.sh (hunk 2 above)
Comment (hanke) [inline tools/sshserver_prepare_for_publish.sh]: AFAIK that is not needed (or useful) for assignments.

Comment (Alexqw) [inline tools/sshserver_prepare_for_publish.sh]: Field splitting is not done on variable assignment, thus quotes are not needed.

**Comment by yarikoptic** [comment]
included within #199

**Pull request closed without merging**